src/common/appmain.cpp

```diff
@@ -233,6 +233,7 @@ void app_run(void) {
             loop();
         }
         marlin_server::loop();
+        buddy::metrics::RecordUptime(); // Pa465
```

Comment:
1. What does this comment stand for?
2. Why do we need to call this function from two places?